src/lib.rs

```diff
@@ -970,14 +982,16 @@ impl MatchOptions {
     /// MatchOptions {
     ///     case_sensitive: true,
     ///     require_literal_separator: false,
-    ///     require_literal_leading_dot: false
+    ///     require_literal_leading_dot: false,
+    ///     follow_links: false,
     /// }
     /// ```
     pub fn new() -> MatchOptions {
         MatchOptions {
             case_sensitive: true,
             require_literal_separator: false,
             require_literal_leading_dot: false,
+            follow_links: false,
```

Comment:
I think it's the right call (it matches all shells I tried as well as Python's `glob` module) but a default of `false` would be a breaking change, right?